saige_assoc.py

```diff
@@ -200,123 +238,255 @@ def apply_job_settings(job, job_name: str):
             job.cpu(cpu)
 
 
+def run_fit_null_job(
+    null_output_path: str,
+    pheno_file: str,
+    cov_col_list: str,
+    sample_cov_col_list: str,
+    sample_id_pheno: str,
+    plink_path: str,
+    pheno_col: str,
+    skip_vre: str = 'FALSE',
+    is_cov_transform: str = 'TRUE',
+):
+    """
+    Check if the output file already exists;
+        if it does, read into a resource group
+        if it doesn't, run the command and pass the resource group back
+    Args:
+        null_output_path ():
+        pheno_file ():
+        cov_col_list ():
+        sample_cov_col_list ():
+        sample_id_pheno ():
+        plink_path ():
+        pheno_col ():
+
+    Returns:
+        Tuple: (Job | None, ResourceGroup)
+
+    """
+    if to_path(null_output_path).exists():
+        return None, get_batch().read_input_group(
+            **{
+                'rda': f'{null_output_path}.rda',
+                'varianceRatio': f'{null_output_path}.varianceRatio.txt',
```

Comment:
is this definitely the right extension?

Reply:
I think so! 

See for example the example output in the SAIGE-QTL repo: https://github.com/weizhou0/qtl/blob/main/extdata/output/example_binary_1.1.4.varianceRatio.txt

And also the documentation:
>  less -S ./output/nindep_100_ncell_100_lambda_2_tauIntraSample_0.5_gene_1.varianceRatio.txt
https://weizhou0.github.io/SAIGE-QTL-doc/docs/step1.html

What makes you think it is not?